# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/css/css-transforms/transform-box/reference/svgbox-rect-dasharray-ref.html
# Commit: e215151d2549
# Full Hash: e215151d2549581c23a1d3c8b53d8899f9776114
# Author: Boris Chiou <boris.chiou@gmail.com>
# Date: 2023-08-23 04:21:33
# Regressor Bug: 1819464
# File Overlap Count: 1
# Description:
#   Bug 1819464 - Part 5: Support stroke-box (for SVG layout). r=emilio
#   
#   We should avoid using the SVG shape elements, e.g. path, polygon, which may not
#   have simple bounds, and its stroke bbox contribution may depends on the
#   platform and the drawing backend if using Moz2D. So for now it'd be better to
# ==============================================================================

diff -r 803ea4d17810 -r e215151d2549 testing/web-platform/tests/css/css-transforms/transform-box/reference/svgbox-rect-dasharray-ref.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/tests/css/css-transforms/transform-box/reference/svgbox-rect-dasharray-ref.html	Tue Aug 22 19:49:04 2023 +0000
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<style>
+#target {
+  fill: green;
+  stroke: black;
+  stroke-width: 20;
+  stroke-dasharray: 100 50;
+}
+</style>
+<svg width="400" height="300">
+  <rect id="target" width="50" height="100" x="50" y="80"/>
+</svg>